Sources/Commands/Utilities/APIDigester.swift

```diff
@@ -192,11 +193,24 @@ public struct SwiftAPIDigester {
         args += try buildPlan.createAPIToolCommonArgs(includeLibrarySearchPaths: false)
         args += ["-module", module, "-o", outputPath.pathString]
 
-        try runTool(args)
+        let result = try runTool(args)
 
         if !self.fileSystem.exists(outputPath) {
-            throw Error.failedToGenerateBaseline(module)
+            throw Error.failedToGenerateBaseline(module: module)
         }
+
+        try self.fileSystem.readFileContents(outputPath).withData { data in
+            if let jsonObject = try JSONSerialization.jsonObject(with: data, options: []) as? [String:Any] {
+                guard let abiRoot = jsonObject["ABIRoot"] as? [String:Any] else {
+                    throw Error.failedToValidateBaseline(module: module)
+                }
+
+                guard let symbols = abiRoot["children"] as? NSArray, symbols.count > 0 else {
+                    throw Error.noSymbolsInBaseline(module: module, toolOutput: try result.utf8Output())
```

Comment:
cc @nkcsgexi, does seem like a valid way to check whether the produced file contains any symbols? Is there a better of figuring out whether `swift-api-digester` failed?

Reply:
Can we pass down `-compiler-style-diags` to swift-api-digester invocations? That flag should give us the correct return value.

Reply:
Still seeing a 0 return value here, even when passing `-compiler-style-diags`